docs/docs/ui.md

```diff
@@ -124,4 +124,5 @@ The `<ModalView />` component renders a chat message feed in a modal window. It
 | `initialMessageText`             | `string?`      | The text content of an initial assistant message at the top of the message feed.               | If no text is specified, then no message is shown.                                           |
 | `inputBarPlaceholder`            | `string?`      | The placeholder text shown when the input bar is empty.                                        | If not specified, the input bar uses default placeholders.                                   |
 | `inputBottomText`                | `string?`      | Text that appears immediately below the input bar.                                             | If not specified, no bottom text is shown.                                                   |
+| `shouldUseHotkey`                | `boolean?`     | If `true`, the chat window will show on hotkey press (`/`)                                     | `false`                                                                                      |
```

Comment:
perhaps rename to 'shouldUseSlashHotkey` to be more specific. 

alternatively, could make sense to let the user define what the hotkey is. then user could set it to `"/"`, `"s"`, etc

Reply:
+1 for customizable hotkey. I think ideally hotkey stuff would be handled outside of our library, but we can add it as a convenience for this case.

Taking a step back, I think the most composable pattern is to present this as a new type of trigger component, a `HotkeyTrigger`. You can use this alongside the input bar trigger inside of the chatbot provider.

```tsx
function MyApp() {
  return (
    <Chatbot>
      <HotkeyTrigger onKey="/" />
      <DevCenterInputBarTrigger />
    </Chatbot>
  )
}
```

Reply:
Let's quietly ship `shouldUseSlashHotkey` as a Chatbot prop and then remove it in favor of a more customizable trigger.